storage/bootstrap.go

```diff
@@ -89,18 +91,6 @@ func (bsm *bootstrapManager) cutoverTime(writeStart time.Time) time.Time {
 	return writeStart.Add(bufferFuture)
 }
 
-// flushTime determines the time we can flush data for after the bootstrap
-// process is finished (i.e., all the shards have bootstrapped).
-func (bsm *bootstrapManager) flushTime(writeStart time.Time) time.Time {
-	// NB(xichen): when bootstrap process is finished, we should have bootstrapped
-	// everything until writeStart + bufferFuture, which means the current time is
-	// at least writeStart + bufferFuture + bufferPast.
-	rops := bsm.opts.GetRetentionOptions()
-	bufferPast := rops.GetBufferPast()
-	bufferFuture := rops.GetBufferFuture()
-	return writeStart.Add(bufferFuture).Add(bufferPast)
-}
-
 // NB(xichen): Bootstrap must be called after the server starts accepting writes.
```

Comment:
Yeah so the issue here is that it needs to be called after server starts accepting writes (not after the database is open) so we don't miss writes. 

Maybe we can have the server push a notification to a channel after it starts up and have `Bootstrap` listen to that channel before it starts bootstrapping?